src/frontend/src/components/dappList.ts

```diff
@@ -0,0 +1,107 @@
+import { html, TemplateResult } from "lit-html";
+
+export type DappDescription = {
+  id: string;
+  name: string;
+  oneLiner?: string;
+  link: string;
+  derivationOrigin?: string;
```

Comment:
why is the derivation origin needed here?

Reply:
@peterpeterparker I worked with the data that come from @frederikrothenberger json. It sometimes has this key and sometimes not.

Reply:
@nmattia the initial idea was that @frederikrothenberger would add it separately. But that changed now, so I will include the JSON file.